pagopa/pn-frontend
# Pn 2873 fe gestire il retry after in caso di legal facts e atti finiti in glacier

## Short description
Added retryAfter handling for legal facts documents download

## List of changes proposed in this pull request
- Added new component TimedMessage, it features message showing for X milliseconds
- Edited redux files about notifications and both detailed notification pages in PA and PF, including tests.

## How to test
PA or PF: select a notification where a legal fact document is still in Glacier. When you click to download the document, it should appear a message about wait and retry later for X seconds where X is taken from retryAfter response object from legal fact download API. After timeout, the page will reload.
